src/NexyCrypt.php

```diff
@@ -320,7 +320,7 @@ private function signedPostRequest($uri, array $payload)
      *
      * @return ResponseInterface
      */
-    private function request($method, $uri, array $options = ['verify' => false])
+    private function request($method, $uri, array $options = ['verify' =>  __DIR__.'/cacert.pem'])
```

Comment:
The problem is still the same: Passing a array as default argument.

BTW, if you pass a path, the option name should be changed to be more consistent.